exporter/datadogexporter/translate_traces_test.go

```diff
@@ -273,6 +273,13 @@ func TestBasicTracesTranslation(t *testing.T) {
 	// ensure a duration and start time are calculated
 	assert.NotNil(t, datadogPayload.Traces[0].Spans[0].Start)
 	assert.NotNil(t, datadogPayload.Traces[0].Spans[0].Duration)
+
+	// ensure that events tag is set if span events exist and contains structured json fields
+	assert.Contains(t, "start", datadogPayload.Traces[0].Spans[0].Meta["Events"])
```

Comment:
Can we go ahead and do an exact assert against a json string? I think it'll make the test much easier to grok

Reply:
yup makes sense, will update